virtcontainers/network.go

```diff
@@ -417,7 +417,29 @@ func (endpoint *PhysicalEndpoint) Detach(netNsCreated bool, netNsPath string) er
 
 // HotAttach for physical endpoint not supported yet
 func (endpoint *PhysicalEndpoint) HotAttach(h hypervisor) error {
-	return fmt.Errorf("PhysicalEndpoint does not support Hot attach")
+	networkLogger().WithField("endpoint-type", "physical").Info("Attaching endpoint (hotplug)")
```

Comment:
You will need to implement HotDetach as well for physical interface.

Reply:
Implementing `HotDetach()` needs some more rework. It needs the ID of the device to be stored, but the structures don't provide this ability. The hot detach requires more work, and can be handled in a follow up PR.